Documentation/tutorial/intro_rest_api_documentation.md

```diff
@@ -104,6 +117,94 @@ Footer for `microsoft.com/docfx/Contacts/1.6`
 
 If `footer` is set, the content from `footer` will be appended to the last section of the **File** or **API**. It is usually used to define **See Also** or **Additional Resources** for the documentation.
 
+### Tags to organize the sections of APIs
+You can organize the sections of APIs by using tags in Swagger file, following definitions in [Tag Object](http://swagger.io/specification/#tagObject). 
+
+Each API can be specified with one or multiple tags, or not speficied with any tag.
+- If all APIs are *not* tagged, each API will not be included in any sections.
+- If the API is specified with *one* tag only, it will show inside this one tag section.
+- If the API is specified with *multiple* tags, it will show inside multiple tag sections.
+- If some APIs are specified with tags while some other APIs are not, the untagged APIs will be organized into one auto generated `Other apis` section.
+
+Specific bookmark could be added to tag section using `x-bookmark-id`, which is Swagger schema extensions following [Specification Extensions](http://swagger.io/specification/#vendorExtensions). If no `x-bookmark-id` is specified, `name` of the tag will be the default bookmark.
+
+For example, the following swagger file defines `Basic` and `Advanced` tags.
+1. Sections in the layout:
+   - `set_contacts` API is tagged with `Advanced` only, then it will only show inside `Advanced` tag section.
```

Comment:
`Advanced` is the name of the tag section...

---
In reply to: [65512141](https://github.com/dotnet/docfx/pull/349#discussion_r65512141) [](ancestors = 65512141)